### PR TITLE
Apply responsive content box and padding to more pages for consistency

### PR DESCRIPTION
**Pull request by davesteinberg**
This builds on #94, using its common `ContentBox` component to apply consistent page widths and responsive padding on most other pages in the site. I haven't dealt with the single-story page yet, since there's already an issue about making it responsive.

### CODE REVIEW

File: components/common/FloatingRibbon.tsx [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+      pos="sticky"
       bottom={0}
       left={0}
       right={0}
-      p={4}
-      borderTopRadius='8px'
-      boxShadow='0px -4px 4px rgba(0, 0, 0, 0.1)'
-      bg='white'
+      py={4}
+      px={RESPONSIVE_PADDING}
```
Comment (davesteinberg) [inline components/common/FloatingRibbon.tsx:12]: Lots of reformatting by prettier in this one. Main change is here: `px={RESPONSIVE_PADDING}`

File: components/common/Nav.tsx [hunk 2]
```diff
@@ -123,7 +126,8 @@ const NavBarContainer = ({ children, ...props }) => {
       justify="space-between"
       wrap="wrap"
       w="100%"
-      p={8}
+      py={4}
+      px={RESPONSIVE_PADDING}
```
Comment (davesteinberg) [inline components/common/Nav.tsx:130]: Main change here: vertical padding to 16px, horizontal padding to 16px (phone) or 32px (tablet/desktop), as per the design.

File: pages/about.tsx [hunk 3]
```diff
@@ -8,14 +9,12 @@ export default function About() {
...
       </Head>
-      <Box w="95%" p={8} pb={24}>
-        <Box pb={4}>
-          <Heading as="h1" size="2xl" pb={4}>
-            About
-          </Heading>
-        </Box>
-        <Box pb={7}>
-          <Heading as="h2" size="l" pb={4}>
+      <ContentBox py>
```
Comment (davesteinberg) [inline pages/about.tsx:12]: Main change here: adopted `ContentBox` for standard page width and responsive padding. I also decreased the padding between sections below to better balance with the padding.

File: pages/new.tsx [hunk 4]
```diff
@@ -1,25 +1,22 @@
...
-            action in Ontario. We value your privacy.
-          </Text>
-          <Text pt={4} fontWeight="bold" color="#55099D">
-            <Link href="/faq">Read our FAQ</Link>
-          </Text>
-        </Box>
-        <StoryForm />
-      </Box>
-    </div>
+    <ContentBox py>
```
Comment (davesteinberg) [inline pages/new.tsx:8]: Again, adopted `ContentBox` for standard page width and responsive padding, also removed the extra nested `Box`.

**Review by mwickett — APPROVED** [review]

**Pull request merged**